cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -22,6 +22,7 @@ import (
 	"time"
 
 	dmysql "github.com/go-sql-driver/mysql"
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
Note the license of `hashicorp/golang-lru` is MPL, we need to confirm whether it is appropriate to use in tiflow repo.

Reply:
agree, seems need to seek for advice from the legal colleague